src/app/api/checkout/route.ts

```diff
@@ -0,0 +1,66 @@
+// NOTE: NextRequest and NextReponse will only work for middleware, and not for 
+// pages or components, so there is a specific command line for it. If it does 
+// not have a command line above it, then it will cause an error, like: 
+// Imports "NextRequest" are only used as type.
+
+// eslint-disable-next-line
+import { NextRequest, NextResponse } from "next/server";
+import Stripe from 'stripe';
+
+// NOTE: Instead using `require()`, should use `Stripe()`
```

Comment:
Remove this comment; it doesn't add anything. Otherwise you have to `NOTE:` every decision you DIDN'T make and you'll never have time for that